satellite/gracefulexit/endpoint.go

```diff
@@ -178,7 +178,7 @@ func (endpoint *Endpoint) Process(stream pb.SatelliteGracefulExit_ProcessServer)
 	return endpoint.doProcess(stream)
 }
 
-// Process is called by storage nodes to receive pieces to transfer to new nodes and get exit status.
+// Process is called by storage nodes to receive pieceIDs to transfer pieces to new nodes and get exit status.
```

Comment:
nit - while the original comment wasn't 100% accurate, I think these changes make it more confusing. Not sure how to reword it off the top of my head, but I'll try to think of something by my next review of this.

Reply:
okay I'll just remove changes for now